.github/docker/Dockerfile.centreon-web-22.10.0-alma-8

```diff
@@ -0,0 +1,68 @@
+FROM docker.centreon.com/proxy/almalinux:8 AS web_fresh
+
+RUN \
```

Comment:
I guess you do not use heredoc syntax to build it locally, but do not forget to use it when you integrate this build in github workflow

Reply:
@kduret will do, it's just here doc doesn't work on windows for some reasons ^^